### PR TITLE
feat: 5.x Update outputs for worker pools/ips/ids, eviction grace in seconds

### PR DESCRIPTION
**Pull request by devoncrouse**
* Updated worker outputs for instance IDs => IPs where available
```
Outputs:

worker_pool_ips = {
  "example-mode-instance" = {
    "ocid1.instance..." = "10.x.x.x"
  }
  "example-mode-nodepool" = {
    "ocid1.instance..." = "10.x.x.x"
  }
}
```

* Managed node pool cordon & drain eviction grace period in seconds; fixes reported change:
```
  # module.workers.module.workers[0].oci_containerengine_node_pool.workers["..."] will be updated in-place                                                                      
  ~ resource "oci_containerengine_node_pool" "workers" {                                                                                                                                
...
      ~ node_eviction_node_pool_settings {
          ~ eviction_grace_duration              = "PT0S" -> "PT0M"
            # (1 unchanged attribute hidden)
        }
```

### CODE REVIEW

**Review by rppala90 — APPROVED** [review]
lgtm

File: modules/workers/nodepools.tf [hunk 1]
```diff
@@ -80,7 +80,7 @@ resource "oci_containerengine_node_pool" "workers" {
   )
 
   node_eviction_node_pool_settings {
-    eviction_grace_duration              = format("PT%sM", each.value.eviction_grace_duration)
+    eviction_grace_duration              = format("PT%sS", each.value.eviction_grace_duration)
```
Comment (lodotek) [inline modules/workers/nodepools.tf:83]: why was this changed away from `PT%sM` to `PT%sS` ? This is causing a permadiff in RC4.
